src/libraries.sh

```diff
@@ -487,6 +488,13 @@ rlImport() {
     if ! eval $VERIFIER
     then
       rlLogError "rlImport: Import of library $LIBRARY was not successful (callback failed)"
+      if ! declare -F $VERIFIER > /dev/null
+      then
+        rlLogInfo "rlImport: Missing callback function $VERIFIER in lib.sh"
+        local FOO=$(grep -Eo "\w*LibraryLoaded" $LIBFILE) && \
```

Comment:
It would be good first to ignore all comments as it may be found there as well, something like `sed -r 's/#.*//'`.
Also more precise matching could help - `grep -Eo '\w*LibraryLoaded\W*\(|function\W*\w*LibraryLoaded\b'`.
This seems to me quite robust `sed -r 's/#.*//' $LIBFILE | grep -Eo '\w+LibraryLoaded\s*\(|function\W+\w+LibraryLoaded\b' | grep -Eo "\w+LibraryLoaded" | sort | uniq`.
Note that there may be more such functions defined so we should need to pick one or handle an array.